backpack/__init__.py

```diff
@@ -224,13 +228,26 @@ def extend(module: Module, debug: bool = False) -> Module:
     Args:
         module: The module to extend.
         debug: Print debug messages during the extension. Default: ``False``.
+        use_converter: whether to convert the module to a BackPACK-compatible network.
 
     Returns:
         Extended module.
+
+    Raises:
+        NotImplementedError: if trying to use converter without torch>=1.9.0
     """
     if debug:
         print("[DEBUG] Extending", module)
 
+    if use_converter:
+        if TORCH_VERSION_AT_LEAST_1_9_0:
+            module = convert_module_to_backpack(module)
```

Comment:
Q: This recursively converts `module`, so there is no need to pass `use_converter` down the recursion?

Reply:
Yes, exactly.